packages/enzyme/src/ReactWrapper.js

```diff
@@ -1189,6 +1236,28 @@ class ReactWrapper {
   }
 }
 
+/**
+ * A *special* "root" wrapper that represents the component passed as `wrappingComponent`.
+ * It is linked to the primary root such that updates to it will update the primary,
```

Comment:
the comment here seems to contradict the sentence in the docs

Reply:
Yup the docs are wrong. Sorry!